### PR TITLE
{bio}[intel/2015a] Some bioinformatics tools (REVIEW)

### PR DESCRIPTION
**Pull request by gjbex**
- beagle-lib
- BEDTools
- BWA
- Bowtie
- Bowtie2
- MrBayes

### CODE REVIEW

**Comment by gjbex** [comment]
Test report by @gjbex
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
Linux red hat enterprise linux computenode 6.4, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.6
See https://gist.github.com/8715adc48ca34ad9891b for a full test report.

File: easybuild/easyconfigs/m/MrBayes/MrBayes-3.2.4-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name = 'MrBayes'
+version = '3.2.4'
+
+homepage = 'http://mrbayes.csit.fsu.edu'
+description = "MrBayes is a program for the Bayesian estimation of phylogeny."
+
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/MrBayes/MrBayes-3.2.4-intel-2015a.eb]: drop a empty line?

File: easybuild/easyconfigs/b/beagle-lib/beagle-lib-20120124-intel-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+# there is no tarball provided, only SVN checkout through:
+# svn checkout http://beagle-lib.googlecode.com/svn/trunk/ beagle-lib
+sources = [SOURCE_TGZ]
+
+dependencies = [('Java', '1.8.0_31', '', True)]
+
+preconfigopts = './autogen.sh && '
```
Comment (wpoely86) [inline easybuild/easyconfigs/b/beagle-lib/beagle-lib-20120124-intel-2015a.eb]: `autoconf` & `automake` should be builddependencies for this...

**Comment by wpoely86** [comment]
ok to test

File: easybuild/easyconfigs/b/BEDTools/BEDTools-2.22.1-intel-2015a.eb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+description = """The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps
+ and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF,
+ and SAM/BAM."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['https://github.com/arq5x/bedtools2/archive/']
+sources = ['v%(version)s.tar.gz']
+
+buildopts = 'CXX="$CXX"'
```
Comment (wpoely86) [inline easybuild/easyconfigs/b/BEDTools/BEDTools-2.22.1-intel-2015a.eb]: is is not picked up automatically?

**Comment by wpoely86** [comment]
retest this please

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 4 out of 6 (6 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/eb9a46e7ed1089ab7df3 for a full test report.

**Comment by boegel** [comment]
```
configure.ac:63: error: possibly undefined macro: AM_DISABLE_STATIC
configure.ac:64: error: possibly undefined macro: AC_PROG_LIBTOOL
configure.ac:65: error: possibly undefined macro: AM_PROG_LIBTOOL
```

We've seen this before, but I can't remember the fix from the top of my head.

@wpoely86, do you remember?

**Comment by wpoely86** [comment]
yes, it's libtool. Add a build dep on the 'autotools' bundle.

File: easybuild/easyconfigs/b/beagle-lib/beagle-lib-20120124-intel-2015a.eb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+# there is no tarball provided, only SVN checkout through:
+# svn checkout http://beagle-lib.googlecode.com/svn/trunk/ beagle-lib
+sources = [SOURCE_TGZ]
+
+dependencies = [
+    ('Java', '1.8.0_31', '', True)
+]
+builddependencies = [
+    ('Automake', '1.15', '', ('GCC', '4.9.2')),
```
Comment (wpoely86) [inline easybuild/easyconfigs/b/beagle-lib/beagle-lib-20120124-intel-2015a.eb]: this is not enough. ALWAYS add the `autotools` bundle when you want to run `autogen.sh`.

Comment (boegel) [inline easybuild/easyconfigs/b/beagle-lib/beagle-lib-20120124-intel-2015a.eb]: @gjbex: please fix?

File: easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.2.4-intel-2015a.eb [hunk 5]
```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Modified by: Robert Schmidt <rjeschmi@gmail.com>
+# Ottawa Hospital Research Institute - Bioinformatics Team
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.2.4-intel-2015a.eb]: here is an example where my type of header is more readable, this situation has been considered.
You know where to find it ;-)
btw. I'd like at least the original author, Cedric, to be also in the list just as a means of thanks...

File: easybuild/easyconfigs/b/beagle-lib/beagle-lib-20120124-intel-2015a.eb [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+ phylogenetics packages."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+# there is no tarball provided, only SVN checkout through:
+# svn checkout http://beagle-lib.googlecode.com/svn/trunk/ beagle-lib
+sources = [SOURCE_TGZ]
+
+dependencies = [
+    ('Java', '1.8.0_31', '', True)
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/b/beagle-lib/beagle-lib-20120124-intel-2015a.eb]: plzzzz: 1.7.0_80 ;-) 
(or, if there is a technical need for 1.8, understood, just comment it!)

File: easybuild/easyconfigs/b/beagle-lib/beagle-lib-20120124-intel-2015a.eb (hunk 4 above)
Comment (fgeorgatos) [inline easybuild/easyconfigs/b/beagle-lib/beagle-lib-20120124-intel-2015a.eb]: @gjbex: ping :)

File: easybuild/easyconfigs/b/beagle-lib/beagle-lib-20120124-intel-2015a.eb (hunk 6 above)
Comment (boegel) [inline easybuild/easyconfigs/b/beagle-lib/beagle-lib-20120124-intel-2015a.eb]: 1.7 is deprecated, we shouldn't keep dragging it along

What we really need is flexible dep versions, and a good way of dealing with them...

**Comment by migueldiascosta** [comment]
@gjbex, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**